DPG/Tasks/AOTTrack/qaEfficiency.cxx

```diff
@@ -1714,7 +1714,7 @@ struct QaEfficiency {
   SliceCache cache;
   Preslice<o2::aod::Tracks> perCollision = o2::aod::track::collisionId;
   Preslice<o2::aod::McParticles> perCollisionMc = o2::aod::mcparticle::mcCollisionId;
-  void processMC(o2::aod::McCollision const&,
+  void processMC(o2::aod::McCollisions const&,
```

Comment:
big question: in this way, I am afraid that you loop over `collisions` for every `McCollision`, because reconstructed collisions are not grouped with MC collisions, am I right?

Reply:
@njacazio

Reply:
I think that's the intended way, I loop over all collisions in the DF

Reply:
Still not clear to me. If you have 1000 reco collisions, and each of them correspond to a reco collision, then w/o the grouping you have 1000 /*MC collisions*/ TIMES 1000 /*reco. collisions*/, because for each MC collision you loop over the full reco. coll. table of the DF. Am I right?

Reply:
this grouping is done thanks to the index in the `McCollisionLabel`, sorry @njacazio for the noise. Fine for me then.

Reply:
np